Dockerfile

```diff
@@ -14,7 +14,13 @@ COPY . /app
 
 RUN python -m pip install -U pip
 
-RUN pip install -r /app/requirements.txt
+RUN python -m pip install pip-tools
+
+# update and install dev requirements
+RUN cd /app && ./update_install_dev_reqs.sh
 
 # for running the workflow
-RUN python -m ipykernel install --user --name python38myenv
\ No newline at end of file
+RUN python -m ipykernel install --user --name python38myenv
+
+# Uncomment to run jupyterlab when container is launched:
+# RUN jupyter-lab --allow-root --config .devcontainer/jupyter-server-config.py 
```

Comment:
I'm not quite sure why this is. You can put CMD as the default, but *optional* startup command. It can be easily overridden if need be. Otherwise I'd remove this and document alternative usage elsewhere.

Reply:
I'll remove this. It's documented in the local install instructions, and I believe people who are familiar with container development will run into the config file when they need to.